src/referencePackages/Directory.Build.props

```diff
@@ -1,7 +1,13 @@
 <Project>
 
   <Import Project="..\..\Directory.Build.props" />
-  
+
+  <PropertyGroup>
+    <CustomizationsPropsPath>$(MSBuildProjectDirectory)\$(CustomizationsPropsFile)</CustomizationsPropsPath>
+  </PropertyGroup>
```

Comment:
What's the reason for not also automatically including Customizations.cs if that file exists?

Reply:
Discussed in your other comment.